about/about.html

```diff
@@ -0,0 +1,127 @@
+                     </p>
+                      <img src="https://img.icons8.com/windows/64/null/github-squared.png" onclick="location.href=' https://github.com/arcanistzed'"/>
+                      <img src="https://img.icons8.com/fluency/64/null/linkedin-circled.png"onclick="location.href=' https://linkedin.com/in/arcanistzed'"/>
+                    </div>
+                </div>
+            </div>
+
+      <div class="column">
+          <div class="card">     
+                <div class="container">
+                  <h3>Mehdi Dahamna</h3>
+                  <img src="https://img.icons8.com/windows/64/null/github-squared.png"/>
+                  <img src="https://img.icons8.com/fluency/64/null/linkedin-circled.png"/>
+                    <p>mehdi@example.com</p>
+                  </div>
+              </div>
+          </div>
+            
+        <div class="column">
+            <div class="card"> 
+              <img src="kylelabelle.PNG" style="width:100%"/>   
+                <div class="container">
+                    <h3>Kyle Labelle</h3>
+                      <p>klabe099@uottawa.ca</p>
+                      <img src="https://img.icons8.com/windows/64/null/github-squared.png" onclick="location.href='https://github.com/kyle-labelle'"/>
+                      <img src="https://img.icons8.com/fluency/64/null/linkedin-circled.png"onclick="location.href='https://www.linkedin.com/in/kyle-labelle/'"/>
+                    </div>
+                </div>
+            </div>
+            
+          <div class="column">
+              <div class="card">
+                  <img src="nimetkurnaz.png" style="width:100%"/>
+                  <div class="container">
+                      <h3>Nimet Kurnaz</h3>
+                      <p>nkurn079@uottawa.ca</p>
+                      <img src="https://img.icons8.com/windows/64/null/github-squared.png" onclick="location.href='https://github.com/Nimetkurnaz/'"/>
+                      <img src="https://img.icons8.com/fluency/64/null/linkedin-circled.png"onclick="location.href='https://www.linkedin.com/in/nimetkurnaz/'"/>
+                      </div>
+                  </div>
+              </div>
+            
+          <div class="column">
+              <div class="card"> 
+                <img src="peterbou-farah.PNG" style="width:100%"/> 
+                  <div class="container">
+                      <h3>Peter Bou-Farah</h3>
+                      <p>pbouf098@uottawa.ca</p>
+                      <img src="https://img.icons8.com/windows/64/null/github-squared.png" onclick="location.href='https://github.com/peter-bf/'"/>
+                      <img src="https://img.icons8.com/fluency/64/null/linkedin-circled.png"onclick="location.href='https://www.linkedin.com/in/peter-bou-farah/'" /> 
+                      </div>
+                  </div>
+              </div>
+
+            </div>
+	
+      <div id="footer">
+
+          <div>
+            <b></b><br />
```

Comment:
These `<br>` tags should be removed in favor of a CSS solution as discussed in our meeting.

Reply:
`<b>` tag is invalid